proto/query.proto

```diff
@@ -562,6 +562,36 @@ message BeginExecuteBatchResponse {
   int64 transaction_id = 3;
 }
 
+// MessageStreamRequest is the request payload for MessageStream.
+message MessageStreamRequest {
+  vtrpc.CallerID effective_caller_id = 1;
+  VTGateCallerID immediate_caller_id = 2;
+  Target target = 3;
+  // name is the message table name.
+  string name = 4;
+}
+
+// MessageStreamResponse is a response for MessageStream.
+message MessageStreamResponse {
+  QueryResult result = 1;
+}
+
+// MessageAckRequest is the request payload for MessageAck.
+message MessageAckRequest {
+  vtrpc.CallerID effective_caller_id = 1;
+  VTGateCallerID immediate_caller_id = 2;
+  Target target = 3;
+  // name is the message table name.
+  string name = 4;
+  repeated Value ids = 5;
+}
+
+// MessageAckResponse is the response for MessageAck.
+message MessageAckResponse {
+  // count is the number of messages acked.
+  int64 count = 1;
```

Comment:
This would be the same as this one, how about renaming it to 'rows_affected'?

message QueryResult {
...
  uint64 rows_affected = 2;
...
}